codechecker_common/report.py

```diff
@@ -29,6 +29,12 @@
 LOG = get_logger('report')
 
 
+def str_to_hash(string_to_hash, errors='ignore'):
+    """ Encodes the given string and generates a hash from it. """
+    string_hash = string_to_hash.encode(encoding="utf-8", errors=errors)
+    return hashlib.md5(string_hash).hexdigest()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4587795)